packages/collaboration-extension/src/filebrowser.ts

```diff
@@ -160,6 +161,42 @@ export const defaultFileBrowser: JupyterFrontEndPlugin<IDefaultFileBrowser> = {
   }
 };
 
+/**
+ * The default collaborative drive provider.
+ */
+export const logger: JupyterFrontEndPlugin<void> = {
+  id: '@jupyter/collaboration-extension:logger',
+  description: 'A logging plugin for debugging purposes.',
+  autoStart: true,
+  requires: [ITranslator],
+  optional: [],
```

Comment:
You should try to request as few as possible token.

```suggestion
  optional: [ITranslator],
```